### DISCUSSION TITLE
Vercel duplicates my home page when I visit using my own domain

### DISCUSSION THREAD

**Discussion by ArthurDanjou** [nuxt/nuxt#22703] [Questions]
I am using Nuxt v3.6.5 and I deploy my website to Vercel.
The deployment is correct.
When I visit https://dev.arthurdanjou.fr/, my home page is duplicated under the current page, by vercel and I have small details bugged and when I visit https://website-v2-arthurdanjou.vercel.app/, every things are ok.
I tried to deploy to node, all work correctly.
You can check my code here: https://github.com/ArthurDanjou/website-v2

**Comment by elrumo** [comment]
The exact same is happening to me, it's so annoying. Did you manage to figure out what was happening? After searching online for hours I still can't figure it out and it's driving me nuts.
